tests/test_dask_distance.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from __future__ import absolute_import
+from __future__ import absolute_import, division
```

Comment:
Technically we don't need this for this change any more. However it is a good idea to add it anyways as we do make use of division in this file.